.ruff.toml

```diff
@@ -0,0 +1,48 @@
+# Assume Python 3.11
+target-version = "py311"
```

Comment:
we actually test and support 3.10/3.11 and 3.12, so that needs to be taken into account (so the github build workflow, maybe it can be integrated there somehow?)

Reply:
I assumed that the lower supported version was 3.11. I'll change that config to `py310`, so Ruff [know which upgrade suggestions](https://docs.astral.sh/ruff/settings/#target-version) can be safely applied.